### ISSUE TITLE
Language Model Training: Early Stopping

### ISSUE THREAD

**Issue by l0rn0r** [flairNLP/flair#2664]
I have to optimize my hyperparameters for the language model training.
Atm after 144 of 12'443 splits in epoch 1, the loss value & ppl are not decreasing anymore - but the training goes on (got to 1'392 till now).

Is there any way to early stop the language model training (except `max_epochs`)?

Since merge #871 the min learning rate in [`flair/trainers/trainer.py`](https://github.com/flairNLP/flair/blob/master/flair/trainers/trainer.py) can be set as a parameter. Didn't find anything like that in [`language_mode_trainer.py`](https://github.com/flairNLP/flair/blob/master/flair/trainers/language_model_trainer.py).

Maybe some parameter of an optimizer?
Or does it not make sense to stop a language model training early?
